lib/taskana-core/src/test/java/acceptance/classification/CreateClassificationAccTest.java

```diff
@@ -1,297 +1,374 @@
-    assertThatThrownBy(() -> CLASSIFICATION_SERVICE.createClassification(classification))
-        .isInstanceOf(InvalidArgumentException.class);
+  void should_ThrowException_TryingToCreateClassificationWithExplicitId() {
+    String explicitId = "EXPLICIT ID";
+
+    ClassificationImpl classification =
+        (ClassificationImpl) classificationService.newClassification("KeyErrCreation", "", "TASK");
+    classification.setId(explicitId);
+
+    assertThatThrownBy(() -> classificationService.createClassification(classification))
+        .isInstanceOf(InvalidArgumentException.class)
+        .hasMessage("ClassificationId should be null on creation");
+  }
+
+  @WithAccessId(user = "taskadmin")
+  @WithAccessId(user = "user-1-1")
+  @TestTemplate
+  void should_ThrowException_When_UserRoleIsNotAdminOrBusinessAdmin() {
+    Classification classification =
+        classificationService.newClassification("KeyErrCreation", "", "TASK");
+
+    assertThatThrownBy(() -> classificationService.createClassification(classification))
+        .isInstanceOf(NotAuthorizedException.class)
+        .hasMessage(
+            String.format(
+                "Not authorized. The current user '%s' is not member of role(s) '[BUSINESS_ADMIN,"
+                    + " ADMIN]'.",
+                taskanaEngine.getCurrentUserContext().getUserid()));
   }
 
   @WithAccessId(user = "businessadmin")
   @Test
-  void testCreateClassificationWithExplicitId() {
-    ClassificationImpl classification =
-        (ClassificationImpl) CLASSIFICATION_SERVICE.newClassification("Key0818", "", "TASK");
-    classification.setId("EXPLICIT ID");
-    assertThatThrownBy(() -> CLASSIFICATION_SERVICE.createClassification(classification))
-        .isInstanceOf(InvalidArgumentException.class);
+  void should_ThrowException_WhenClassificationWithKeyAlreadyExisting() throws Exception {
+    String existingKey = "Key4";
+    DefaultTestEntities.defaultTestClassification()
+        .key(existingKey)
+        .buildAndStore(classificationService);
+
+    Classification classification =
+        classificationService.newClassification(existingKey, "DOMAIN_A", "TASK");
+
+    assertThatThrownBy(() -> classificationService.createClassification(classification))
+        .isInstanceOf(ClassificationAlreadyExistException.class)
+        .hasMessageContaining(
+            String.format(
+                "A Classification with key '%s' already exists in domain 'DOMAIN_A'.",
+                existingKey));
   }
 
   @WithAccessId(user = "businessadmin")
   @Test
-  void should_BeAbleToCreateNewClassification_When_ClassificationCopy() throws Exception {
-    ClassificationImpl oldClassification =
-        (ClassificationImpl) CLASSIFICATION_SERVICE.getClassification("T2100", "DOMAIN_B");
-    Classification newClassification = oldClassification.copy("T9949");
+  void should_CreateNewClassification_When_ClassificationCopy() throws Exception {
```

Comment:
After discussing this with @gitgoodjhe we agreed on removing this test because copy already covers it.

Reply:
alright :)